test/downstream/delaydiffeq.jl

```diff
@@ -31,3 +31,20 @@ DDEProblemLibrary.importddeproblems()
         @test sol[1, :] ≈ sol_scalar.u
     end
 end
+
+τ = 1.0
+function lotka_volterra!(du,u,h,p,t)
+    🐰, 🐺 = u
+    🕥🐰 = h(p,t-τ;idxs=1)
```

Comment:
Suboptimal performance is not relevant in the tests, I assume, but maybe it would still be good to change this line to avoid that users copy it and close over non-constant global variables (I guess OrdinaryDiffEq seems like a respective source).

Reply:
good call.